mealie/routes/recipe/recipe_crud_routes.py

```diff
@@ -103,35 +103,39 @@ def get_recipe_zip_token(self, slug: str):
         return RecipeZipTokenResponse(token=create_recipe_slug_token(slug))
 
     @router_exports.get("/{slug}/exports", response_class=FileResponse)
-    def get_recipe_as_format(self, slug: str, template_name: str, temp_dir=Depends(temporary_dir)):
+    def get_recipe_as_format(self, slug: str, template_name: str):
         """
         ## Parameters
         `template_name`: The name of the template to use to use in the exports listed. Template type will automatically
         be set on the backend. Because of this, it's important that your templates have unique names. See available
         names and formats in the /api/recipes/exports endpoint.
 
         """
+        temp_dir = get_temporary_dir()
         recipe = self.mixins.get_one(slug)
         file = self.service.render_template(recipe, temp_dir, template_name)
-        return FileResponse(file)
+        return FileResponse(file, background=BackgroundTask(rmtree, temp_dir))
```

Comment:
Since this needs to unlink after the file is served you can use `get_temporary_path(auto_unlink=False)` here (see above)